### PR TITLE
21 project new form

### PR DESCRIPTION
**Pull request by KaiHotz**
Refactored some stuff and also resolved the email add and removal, 
Due to the refactor of several components it would be nice to merge this into master to avoid merge conflicts, also wanted to test the pull request :)
captura de pantalla 2017-10-22 a la s 11 36 27

### CODE REVIEW

File: assets/js/components/projects/ProyectsTable.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
```
Comment (lorenzosinisi) [inline assets/js/components/projects/ProyectsTable.js]: Please fix the typo Proyect => Project

File: assets/js/components/projects/ProyectsTable.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import { withRouter } from 'react-router-dom'
+import { compose } from 'recompose'
+import { withStyles } from 'material-ui/styles'
+import Table, {
+  TableHead,
+  TableRow,
+  TableCell
+} from 'material-ui/Table'
+
+import ProyectsTableRows from './ProyectsTableRows'
```
Comment (lorenzosinisi) [inline assets/js/components/projects/ProyectsTable.js]: Please fix the typo Proyect => Project

File: assets/js/components/projects/ProyectsTable.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+ProyectsTable.propTypes = {
+  classes: PropTypes.object.isRequired,
+  data: PropTypes.array
+}
+
+export default compose(
+  withRouter,
+  withStyles(styles)
```
Comment (lorenzosinisi) [inline assets/js/components/projects/ProyectsTable.js]: Please fix the typo Proyect => Project (and check other typos in this file of the same type)

File: assets/js/components/projects/ProyectsTableRows.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import Button from 'material-ui/Button'
+import {
+  TableBody,
+  TableRow,
+  TableCell
+} from 'material-ui/Table'
+
+const ProyectsTableRows = ({proyects}) => {
```
Comment (lorenzosinisi) [inline assets/js/components/projects/ProyectsTableRows.js]: Please fix the typo Proyect => Project

File: assets/js/containers/ProjectNew.js [hunk 5]
```diff
@@ -4,16 +4,23 @@ import { compose } from 'recompose'
 import { connect } from 'react-redux'
```
Comment (lorenzosinisi) [inline assets/js/containers/ProjectNew.js]: Please next time include a screenshot for design changes so it makes it easier to review

File: assets/js/containers/ProjectNew.js [hunk 6]
```diff
@@ -30,7 +37,7 @@ class ProjectNew extends Component {
     name: '',
     emails: [
       {
-        address: null
```
Comment (lorenzosinisi) [inline assets/js/containers/ProjectNew.js]: 🎉

File: assets/js/containers/ProjectNew.js [hunk 7]
```diff
@@ -42,66 +49,86 @@ class ProjectNew extends Component {
...
+              label='Project Name'
+              value={name}
+              className={textField}
+              margin='normal'
+              onChange={this.handleChange('name')}
+              />
+          </Grid>
+          <Grid item xs={12} sm={6}>
+            <Typography type='display1'>
+              Multi Select goes Here :
```
Comment (lorenzosinisi) [inline assets/js/containers/ProjectNew.js]: If you have nothing to show show nothing ;)

File: assets/js/containers/ProjectsContainer.js [hunk 8]
```diff
@@ -58,7 +58,7 @@ class ProjectsContainer extends Component {
           <h4 className='card-header'>
             Contributing
           </h4>
-          <TableProjects
+          <ProyectsTable
```
Comment (lorenzosinisi) [inline assets/js/containers/ProjectsContainer.js]: Please fix the typo Proyect => Project

File: assets/js/containers/ShowProjectContainer.js [hunk 9]
```diff
@@ -32,7 +32,7 @@ class ShowProjectContainer extends Component {
   }
 
   render () {
-    const { currentProject, error, loading } = this.props
```
Comment (lorenzosinisi) [inline assets/js/containers/ShowProjectContainer.js]: Why is this removed?

File: assets/js/components/projects/ProyectsTable.js (hunk 1 above)
Comment (KaiHotz) [reply assets/js/components/projects/ProyectsTable.js]: Ok on it , will do , get confused with spanish hahahah

File: assets/js/containers/ShowProjectContainer.js (hunk 9 above)
Comment (KaiHotz) [reply assets/js/containers/ShowProjectContainer.js]: because we are not using it at this moment in the render methode

File: assets/js/containers/ProjectNew.js (hunk 7 above)
Comment (KaiHotz) [reply assets/js/containers/ProjectNew.js]: need this here as placeholder

File: assets/js/containers/ShowProjectContainer.js (hunk 9 above)
Comment (KaiHotz) [reply assets/js/containers/ShowProjectContainer.js]: because we are not using it in the render method for now

File: assets/js/containers/ProjectNew.js (hunk 7 above)
Comment (KaiHotz) [reply assets/js/containers/ProjectNew.js]: i just need this here as a placeholer

File: assets/js/components/projects/ProyectsTable.js (hunk 1 above)
Comment (KaiHotz) [reply assets/js/components/projects/ProyectsTable.js]: done, got confused with spanish hahaha
